docs/api/picker.md

```diff
@@ -15,116 +15,39 @@ import Slots from '@ionic-internal/component-api/v8/picker/slots.md';
 
 import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
-<EncapsulationPill type="scoped" />
+<EncapsulationPill type="shadow" />
 
 A Picker is a dialog that displays a row of buttons and columns underneath. It appears on top of the app's content, and at the bottom of the viewport.
```

Comment:
```suggestion
A Picker disabled one or more columns with options for users to choose from. 
```

Picker isn't a dialog anymore, and it doesn't to appear at the bottom of the viewport. Open to other variants of this, this is just what came to mind first.